### PR TITLE
Doc HA for data formats that have schema.registry

### LINKED ISSUE #1257: Document: feat: HA for schema registry
This issue tracks the documentation update needed for the merged PR #11982.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/11982
Source PR Merged At: 2023-09-05T10:27:50Z

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  Doc HA for data formats that have schema.registry. Edited data formats: All versions of AVRO, plain JSON, Upsert JSON, protobuf

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  https://github.com/risingwavelabs/risingwave/pull/11982

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1257

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/sql/commands/sql-create-source.md [hunk 1]
```diff
@@ -102,6 +102,8 @@ When creating a source, specify the data and encoding formats in the `FORMAT` an
 
 For data in Avro format, you must specify a message and a schema file location. The schema file location can be an actual Web location that is in `http://...`, `https://...`, or `S3://...` format. For Kafka data in Avro, instead of a schema file location, you can provide a Confluent Schema Registry that RisingWave can get the schema from. For more details about using Schema Registry for Kafka data, see [Read schema from Schema Registry](/create-source/create-source-kafka.md#read-schemas-from-schema-registry).
 
+`schema.registry` can accept multiple addresses. RisingWave will request all URLs and return the first successful URL.
```
Comment (tabVersion) [inline docs/sql/commands/sql-create-source.md]: Return the first successful result

File: docs/sql/commands/sql-create-source.md [hunk 2]
```diff
@@ -132,6 +134,8 @@ ENCODE AVRO (
 
 When creating a source from streams in with Debezium AVRO, the schema of the source does not need to be defined in the `CREATE TABLE` statement as it can be inferred from the `SCHEMA REGISTRY`. This means that the schema file location must be specified. The schema file location can be an actual Web location, which is in `http://...`, `https://...`, or `S3://...` format, or a Confluent Schema Registry. For more details about using Schema Registry for Kafka data, see [Read schema from Schema Registry](/create-source/create-source-kafka.md#read-schemas-from-schema-registry).
 
+`schema.registry` can accept multiple addresses. RisingWave will request all URLs and return the first successful URL.
```
Comment (tabVersion) [inline docs/sql/commands/sql-create-source.md]: Ditto

**Review by tabVersion — APPROVED** [review]
Rest LGTM

File: docs/sql/commands/sql-create-source.md [hunk 3]
```diff
@@ -102,6 +102,8 @@ When creating a source, specify the data and encoding formats in the `FORMAT` an
 
 For data in Avro format, you must specify a message and a schema file location. The schema file location can be an actual Web location that is in `http://...`, `https://...`, or `S3://...` format. For Kafka data in Avro, instead of a schema file location, you can provide a Confluent Schema Registry that RisingWave can get the schema from. For more details about using Schema Registry for Kafka data, see [Read schema from Schema Registry](/create-source/create-source-kafka.md#read-schemas-from-schema-registry).
 
+`schema.registry` can accept multiple addresses. RisingWave will request all URLs and return the first successful result.
```
Comment (hengm3467) [inline docs/sql/commands/sql-create-source.md]: "request all URLs" doesn't sound quite right. Maybe "send requests to all URLs" is better.

File: docs/sql/commands/sql-create-source.md [hunk 4]
```diff
@@ -132,6 +134,8 @@ ENCODE AVRO (
 
 When creating a source from streams in with Debezium AVRO, the schema of the source does not need to be defined in the `CREATE TABLE` statement as it can be inferred from the `SCHEMA REGISTRY`. This means that the schema file location must be specified. The schema file location can be an actual Web location, which is in `http://...`, `https://...`, or `S3://...` format, or a Confluent Schema Registry. For more details about using Schema Registry for Kafka data, see [Read schema from Schema Registry](/create-source/create-source-kafka.md#read-schemas-from-schema-registry).
 
+`schema.registry` can accept multiple addresses. RisingWave will request all URLs and return the first successful result.
```
Comment (hengm3467) [inline docs/sql/commands/sql-create-source.md]: ditto.

File: docs/sql/commands/sql-create-source.md [hunk 5]
```diff
@@ -254,6 +262,8 @@ ENCODE JSON [ (
 
 For data in Protobuf format, you must specify a message and a schema location. The schema location can be an actual Web location that is in `http://...`, `https://...`, or `S3://...` format. For Kafka data in Protobuf, instead of providing a schema location, you can provide a Confluent Schema Registry that RisingWave can get the schema from. For more details about using Schema Registry for Kafka data, see [Read schema from Schema Registry](/create-source/create-source-kafka.md#read-schemas-from-schema-registry).
 
+`schema.registry` can accept multiple addresses. RisingWave will request all URLs and return the first successful result.
```
Comment (hengm3467) [inline docs/sql/commands/sql-create-source.md]: Same here.

**Review by tabVersion — APPROVED** [review]

**Pull request merged**